traitsui/testing/tester/_ui_tester_registry/wx/tests/test_control_widget_registry.py

```diff
@@ -58,10 +58,14 @@ def test_is_enabled(self):
         self.widget.Enable(True)
         self.assertTrue(self.good_wrapper.inspect(IsEnabled()))
 
+    def test_is_visible(self):
+        self.widget.Show(True)
+        self.assertTrue(self.good_wrapper.inspect(IsVisible()))
+
```

Comment:
Let's add the test invisible here - like what we have in qt - unless there is a reason we can't.